scripts/createTags.js

```diff
@@ -36,10 +73,47 @@ function main(solutionsDir, outputFile) {
             }
         }
     });
+
+    for (tag in tags){
+        max = 0;
+        for (k of frequence.keys()){
+            if(k.toLowerCase().replace(/\s+/g,"") == tag){
+                if(max < frequence.get(k)){
+                    max = frequence.get(k);
+                    finalTag = k;
+                } 
+            }
+         }
+         tags[finalTag] = tags[tag];
+         if(finalTag != tag)
+         delete tags[tag];     
+    }
+
+    for(t in tags){
+        for (tag in (tags[t])){
+            max = 0;
+            for (k of frequence.keys()){
```

Comment:
These blocks look very similar. Can you move them or parts of them to a function?